lib/stringtheory/src/interning/map.rs

```diff
@@ -587,7 +593,7 @@ impl GenericMapInterner {
 
     /// Returns the total number of bytes in the interner.
     pub fn len_bytes(&self) -> usize {
-        self.state.lock().unwrap().storage.len
+        self.state.lock().unwrap().storage.offset
```

Comment:
Here is where `len_bytes` is used. 
https://github.com/DataDog/saluki/blob/b7f0294306ac737cfda562efd0864b9759008771/lib/saluki-context/src/resolver.rs#L531

I think we actually want this to return the new `len` field instead of `offset`,